com.unity.render-pipelines.universal/Samples~/TeachingSamples/SharedAssets/Scripts/MobileControls.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MobileControls : MonoBehaviour
```

Comment:
What is on these gameobjects exactly? This script does not really handle controls, it only enables two gameobjects

Reply:
Renamed it to MobileControlsToggler